README.md

```diff
@@ -308,6 +308,67 @@ try {
 
 [action attempt]: https://docs.seam.co/latest/core-concepts/action-attempts
 
+### Pagination
+
+Some Seam API endpoints that return lists of resources support pagination.
+Use the `SeamPaginator` class to fetch and process resources across multiple pages.
+
+#### Manually fetch pages with the nextPageCursor
+
+```ts
+const pages = seam.createPaginator(
+  seam.devices.list({
+    limit: 20,
+  }),
+)
+
+const [devices, { hasNextPage, nextPageCursor }] = await pages.firstPage()
+
+if (hasNextPage) {
+  const [moreDevices] = await pages.nextPage(nextPageCursor)
```

Comment:
@phpnode I changed the method names to remove any ambiguity and avoid any conflict with other language reserved keywords. I could have gone with `getFirst()` or `getFirstPage()` etc, but these seems cleaner. Let me know what you think.